### ISSUE TITLE
Describe rules for schema versioning

### ISSUE THREAD

**Issue by dalito** [nfdi4cat/pid4cat-model#27]
[HCA schema versioning](https://github.com/HumanCellAtlas/metadata-schema/blob/master/docs/evolution.md#schema-versioning) guidelines could serve as example.

**Comment by markdoerr** [comment]
Yes, the HCA versioning  provides a  good template - from what I see, we could copy it 1:1

**Comment by dalito** [comment]
`SchemaVer` proposed in a [snowplow blog post](https://snowplow.io/blog/introducing-schemaver-for-semantic-versioning-of-schemas) also matches with the HCA scheme and is also adapted by [openlinaeage](https://github.com/OpenLineage/OpenLineage/blob/main/spec/Versioning.md).

**SchemaVer**: Given a version number MODEL-REVISION-ADDITION, increment the:
- MODEL when you make a breaking schema change which will prevent interaction with any historical data
- REVISION when you make a schema change which may prevent interaction with some historical data
- ADDITION when you make a schema change that is compatible with all historical data
